### ISSUE TITLE
Falsely passing tests when deny rules are in a nested namespace

### ISSUE THREAD

**Issue by bradfreer** [instrumenta/conftest#298]
As of conftest 0.18.1 I am seeing a possible case where 'test' is erroneously passing rules when they are defined in a nested namespace. I have a deny.rego in a package with a nested namespace: 

```
package global.helm

deny[msg] {
  contains(input.services.php.ports[_], "0.0.0.0")
  msg = "Must not listen on 0.0.0.0"
}

deny[msg] {
  msg = sprintf("Found port %s", input.services.php.ports)
}
```

and testing a YAML file: 

```
services:
  php:
    ports:
      - '0.0.0.0/0:80'
```

now running conftest:

```
 $ conftest test -p deny.rego --namespace global docker-compose.yml
2 tests, 2 passed, 0 warnings, 0 failures
```
```
 $ conftest test -p deny.rego --namespace global.helm docker-compose.yml
? - docker-compose.yml - no policies found
0 tests, 0 passed, 0 warnings, 0 failures
```

Finally changing the deny package to 'global': 

```
 $ conftest test -p deny.rego --namespace global docker-compose.yml
FAIL - docker-compose.yml - Must not listen on 0.0.0.0
FAIL - docker-compose.yml - Found port 0.0.0.0/0:80
2 tests, 0 passed, 0 warnings, 2 failures
```

Is my understanding of how this is intended to work off here? I recall this behaving as I expected on an earlier version.

**Comment by flunderveiko** [comment]
i have the same problem, too bad

**Comment by katsew** [comment]
I'm having the same problem.

**Comment by bradfreer** [comment]
Thanks @katsew, I misread that issue and the changelog for 0.18.1 when I read it. Unfortunately re-writing all of my policies is less work than using master.

**Comment by menendezjaume** [comment]
You're right @bradfreer. Also, the release notes for 0.18.2 from 5 days ago include that PR so you should be able to use that binary without this issue.

**Comment by jpreese** [comment]
Resolved via #295 which is included in `v0.18.2` 👍
